rfcs/20211118-support-implicit-concat/README.md

```diff
@@ -0,0 +1,113 @@
+# RFC: Support of implicit concat
+
+## Introduction
+
+This document explores a way of supporting implicit concat. Implicit concat is a
+concat primitive which is done without any additional copies just by storing
+correspondent parts into right spots of final tensor. Such approach allows to
```

Comment:
"into right spots of final" --> "in the right locations of the final"

Reply:
Applied the suggestion, thanks.